cterasdk/edge/network.py

```diff
@@ -176,3 +177,55 @@ def iperf(self, address, port=5201, threads=1, protocol=IPProtocol.TCP, directio
             return task.result.res
         except TaskError as error:
             return error.task.result.res
+
+    def get_static_routes(self):
+        """
+        Get all Static Routes
+        """
+        return self._gateway.get('/config/network/static_routes')
+
+    def add_static_route(self, source_ip, destination_ip_mask):
+        """
+        Set a Static Route
+
+        :param str source_ip: The source IP
+        :param str destination_ip_mask: The destination IP and CIDR block (10.0.0.0/16)
+        """
+        param = Object()
+        param.GwIP = source_ip
+        param.DestIpMask = destination_ip_mask.replace("/", "_")
+        try:
+            res = self._gateway.add('/config/network/static_routes', param)
+            logging.getLogger().info(
+                "Static route updated. %s", {'Source': param.GwIP, 'Destination': destination_ip_mask})
+            return res
+        except CTERAException as error:
+            logging.getLogger().error("Static route creation failed.")
+            raise CTERAException('Static route creation failed', error)
+
+    def remove_static_route(self, destination_ip_mask):
+        """
+        Delete a Static Route
+
+        :param str destination_ip_mask: The destination IP and CIDR block (10.0.0.0/16)
+        """
+        try:
+            ip_mask = destination_ip_mask.replace("/", "_")
```

Comment:
Python3 includes an ipaddress module. Consider using this:
https://stackoverflow.com/questions/45988215/python-how-to-validate-an-ip-address-with-a-cidr-notation-import-socket
This will also raise errors